RealmSwift/SyncSubscription.swift

```diff
@@ -192,15 +214,19 @@ import Realm.Private
 
     /**
      Synchronously performs any transactions (add/remove/update) to the subscription set within the block.
-     This will not wait for the server to acknowledge and see all the data associated with this collection of subscriptions,
-     and will return after committing the subscription transactions.
 
      - parameter block:      The block containing the subscriptions transactions to perform.
      - parameter onComplete: The block called upon synchronization of subscriptions to the server. Otherwise
                              an `Error`describing what went wrong will be returned by the block
      */
+    public func update(_ block: (() -> Void), onComplete: ((Error?) -> Void)? = nil) {
+        rlmSyncSubscriptionSet.update(block, onComplete: onComplete ?? { _ in })
+    }
+
+    /// :nodoc:
+    @available(*, deprecated, renamed: "update", message: "SyncSubscriptionSet write is deprecated, please use `.update` instead.")
```

Comment:
These should either be deprecated and continue to work or unavailable and a fatalError().